wiki/Reporting_bad_behaviour/es.md

```diff
@@ -2,7 +2,7 @@
 no_native_review: true
```

Comment:
This flag should be removed I think, unless you only changed some bits of this article.